src/rules/validJsdocRule.ts

```diff
@@ -336,7 +337,11 @@ class ValidJsdocWalker extends Lint.RuleWalker {
         case 'returns':
           hasReturns = true;
 
-          if (!OPTIONS.requireReturn && !fn.returnPresent && tag.type.name !== 'void' && tag.type.name !== 'undefined') {
+          if (fn.node.modifiers) {
```

Comment:
`isAbstract = Lint.hasModifier(fn.node.modifiers, ts.SyntaxKind.AbstractKeyword`
No need to check for undefined